walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/script.js

```diff
@@ -41,6 +62,14 @@ function handleResponse(response) {
   textPromise.then(addQuoteToDom);
 }
 
+function addGreetingToDom(greeting) {
+  console.log('Adding greeting to dom: ' + greeting);
+
+  const greetingContainer = document.getElementById('greeting-container');
```

Comment:
You can just do `document.getElementById('greeting-container').innerText = greeting;`

Defining a local variable is useful when it improves readability when you end up using the variable in many different places. Here, you use it just once right after declaring it, so it's simpler and clearer to just directly call `.innerText` on `document.getElementById('greeting-container')`.

Reply:
got it.